src/classify.jl

```diff
@@ -0,0 +1,26 @@
+function _w2cp(vec) 
```

Comment:
could inline this or give a clearer name, took me a while to figure out what it was